drivers/char/fb.c

```diff
@@ -16,8 +16,13 @@
 #include <fiwix/fb.h>
 #include <fiwix/pci.h>
 #include <fiwix/mm.h>
+#include <fiwix/mman.h>
+#include <fiwix/fcntl.h>
 #include <fiwix/bios.h>
 
+#define IO_FB_XRES  2       /* TODO(ghaerr): to be removed shortly */
+#define IO_FB_YRES  3
```

Comment:
Please, use TABs for indentation, not spaces.
You can add tabstop=4 in your .vimrc for your use case.

Reply:
Oops, thought I caught all the TABs issues, sorry. Will push a commit with fixes for everything requested. Thanks for the code review!